src/microbe_stage/MicrobeStage.tscn

```diff
@@ -15,11 +18,31 @@
 [ext_resource type="PackedScene" path="res://src/engine/GuidanceLine.tscn" id="75"]
 [ext_resource type="PackedScene" uid="uid://dwkek50fju0fu" path="res://src/microbe_stage/PatchNameOverlay.tscn" id="85"]
 
-[node name="MicrobeStage" type="Node" node_paths=PackedStringArray("heatViewOverlay")]
+[sub_resource type="FastNoiseLite" id="FastNoiseLite_q12wg"]
+frequency = 0.2401
+
+[sub_resource type="NoiseTexture2D" id="NoiseTexture2D_d28l8"]
+noise = SubResource("FastNoiseLite_q12wg")
+
+[sub_resource type="ShaderMaterial" id="ShaderMaterial_pbi5n"]
+resource_local_to_scene = true
+render_priority = -2
+shader = ExtResource("4_ced6l")
+shader_parameter/warmColour = Color(1, 0.359565, 0.239655, 1)
+shader_parameter/coldColour = Color(0.233659, 0.0244511, 0.882812, 1)
+shader_parameter/uvOffset = Vector2(0, 0)
+shader_parameter/heatThreshold = 0.5
+shader_parameter/alphaMultiplier = 0.077
+shader_parameter/noiseMultiplier = 0.099
+shader_parameter/heat = ExtResource("5_jrgee")
+shader_parameter/noise = SubResource("NoiseTexture2D_d28l8")
```

Comment:
These seem like unused resources so you should restart godot editor and then resave this file which should get rid of these.